### DISCUSSION TITLE
Connecting DAX studio to Power BI report where data model is separated

### DISCUSSION THREAD

**Discussion by gulzaar** [DaxStudio/DaxStudio#1166] [Q&A]
I need to know, how to connect to a power BI reports where the data model is separated. The Power BI report is connected via a Power BI dataset.

i had tired but DAX studio was not able to connect on the above scenario.

**Comment by dgosbell** [comment]
If you are using DAX Studio as an external tool it should just work, but there seems to be a bug in the current release of Power BI Desktop and I've sent this to the product team for investigation.

But you should still be able to connect to the separate dataset directly. This document shows all the different ways you can connect https://daxstudio.org/docs/tutorials/getting-connected/
